README.md

```diff
@@ -2,7 +2,11 @@
 
 ### Installation
 
-1. Ensure `hygen` is installed globally. Follow the [quick start guide](http://www.hygen.io/quick-start).
+1. Ensure `hygen` is installed globally.
+-  ```bash
+    hygen --version
+   ```
+   If command not found follow the [quick start guide](http://www.hygen.io/quick-start).
```

Comment:
are we able to do this in such a way that we keep as a dev dependency and always execute via `yarn hygen`?